Lib/test/libregrtest/runtest.py

```diff
@@ -181,7 +181,7 @@ def test_runner():
             else:
                 test_runner()
             test_time = time.time() - start_time
-        post_test_cleanup()
+            post_test_cleanup()
```

Comment:
I don't think that this change is correct. If support.reap_children() is called "too early", since it uses a non-blocking waitpid(). The function should be called as late as possible, so after the context manager completes. Maybe it would deserve a comment.